cla/signing-tutorial.md

```diff
@@ -0,0 +1,24 @@
+# CoatRack CLA Signing Tutorial
```

Comment:
"Tutorial" is not a good name for this, I would call it "instructions". In general I am wondering if this shouldn't be part of the contribution guidelines file? All information in one page seems to be helpful for prospective contributors.

Reply:
Done. I renamed the 'signing tutorial' to 'signing instructions' and merged them with the guidelines in a CONTRIBUTION.md file.